ci-operator/step-registry/gather/OWNERS

```diff
@@ -9,3 +9,6 @@ approvers:
 - ewolinetz
 - csrwng
 - staebler
+- dcbw
```

Comment:
@pecameron I don't think we need the changes to this file? Are SDN team people going to regularly touch this directory, since we now have a network-specific one?

Reply:
changed